src/ucp/core/ucp_ep.c

```diff
@@ -1113,6 +1113,17 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
         }
     }
 
+    /* configuration for min zcopy */
+    config->tag.rndv.min_get_zcopy = 0;
+    for (lane = 0; lane < config->key.num_lanes &&
+                   config->key.rma_bw_lanes[lane] != UCP_NULL_LANE; ++lane) {
+        rsc_index = config->key.lanes[config->key.rma_bw_lanes[lane]].rsc_index;
+        ucs_assert(rsc_index != UCP_NULL_RESOURCE);
```

Comment:
this assertion fails on jenkins..

Reply:
yep... replaced by ```if``` (like in section below)